src/components/SponsorLogos.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const SponsorListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-bottom: 1em;
+`
+
+const SponsorContainer = styled.div`
+  margin: 1em;
+`
+
+const SponsorImage = styled.img`
+  max-width: ${p => p.width}px;
+  transition: transform 0.2s;
+
+  &:hover {
+    transform: scale(1.1);
+  }
+`
+
+const platWidth = 250
+const tierData = {
+  platinum: { rank: 1, width: platWidth },
+  gold: { rank: 2, width: platWidth * 0.9 },
+  silver: { rank: 3, width: platWidth * 0.8 },
+  bronze: { rank: 4, width: platWidth * 0.7 },
```

Comment:
is there a way to make this depend on rank? like `width: platWidth - 0.1 * rank`? not sure if that works for objects though

Reply:
don't think so unless we separate `tierRanks` and `tierWidths` as separate objects. Could put `tierRanks` in `constants.js